app/controllers/users_controller.rb

```diff
@@ -18,9 +18,29 @@ def create
     end
   end
 
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Before filter added, but now that a there is a `resource user:` in routes that takes care of users only being able to edit themselves.